Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -270,6 +271,28 @@ LayoutUnit GridTrackSizingAlgorithm::initialGrowthLimit(const GridTrackSize& tra
     return infinity;
 }
 
+void GridTrackSizingAlgorithm::sizeTrackToFitSingleSpanMasonryGroup(const GridSpan& span, MasonryIndefiniteItems &masonryIndefiniteItems, GridTrack& track)
```

Comment:
There is an extra space here and this should be `MasonryIndefiniteItems& masonryIndefiniteItems`